src/components/specialQuestions/index.tsx

```diff
@@ -12,7 +13,11 @@ type Props = {
 
```

Comment:
```suggestion
const PLAT_RULENAME = 'alimentation . plats'
```

et utiliser la const partout en dessous